OpenSim/Examples/CustomActuatorExample/PistonActuator.h

```diff
@@ -121,38 +112,33 @@ OpenSim_DECLARE_CONCRETE_OBJECT(PistonActuator, ScalarActuator);
     bool getPointsAreGlobal() {return get_points_are_global(); };
 
     // OPTIMAL FORCE
+    /** The force applied by this actuator is its control signal multiplied
+     * by this optimal force. */
     void setOptimalForce(double aOptimalForce);
     double getOptimalForce() const override;
-    // STRESS
-#ifndef SWIG
-    double getStress( const SimTK::State& s ) const override;
+    /** This is the force generated by this actuator divided by its optimal
+     * force. */
+    double getStress(const SimTK::State& s) const override;
     //--------------------------------------------------------------------------
-    // APPLICATION
+    // FORCE INTERFACE
     //--------------------------------------------------------------------------
+    /** Apply the actuator force to frameA and frameB. */
     virtual void computeForce(const SimTK::State& s, 
```

Comment:
We should remove the virtual keyword. This `PistonActuator` is not defining the virtual method it is overriding the virtual method  of the base `Force` so `override` keyword is sufficient.